sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1019,4 +1026,47 @@ private static String getUserAgent() {
         javaVersion,
         "0.2.3" /* TODO get Bigtable client version directly from jar. */);
   }
+
+  /**
+   * A helper function to produce a Cloud Bigtable user agent string.
+   */
+  public static BigtableOptions.Builder addRetryOptions(BigtableOptions.Builder builder) {
```

Comment:
Did you file the bug against cloud-bigtable-client to add cloning or toBuilder to these methods?